build.sbt

```diff
@@ -19,11 +19,11 @@ libraryDependencies ++= Seq(
       "tests",
   "org.apache.spark" %% "spark-catalyst" % sparkVersion % "test" classifier
       "tests",
-  "org.scalatest" %% "scalatest" % "3.0.5" % "test",
+  "org.scalatest" %% "scalatest" % "3.0.8" % "test",
   "com.novocode" % "junit-interface" % "0.11" % "test",
 
   //SQLServer JDBC jars
-  "com.microsoft.sqlserver" % "mssql-jdbc" % "7.2.1.jre8"
+  "com.microsoft.sqlserver" % "mssql-jdbc" % "8.2.1.jre8"
```

Comment:
why v8.2 for JDBC driver? Did 7.2 give some issue or was this just alignment to latest.

Reply:
Spark 3 is JDK 11, you need to use
```
// https://mvnrepository.com/artifact/com.microsoft.sqlserver/mssql-jdbc
libraryDependencies += "com.microsoft.sqlserver" % "mssql-jdbc" % "8.4.1.jre11"
```